pkg/motor/create_account.go

```diff
@@ -105,7 +105,15 @@ func (mtr *motorNodeImpl) CreateAccountWithKeys(request mt.CreateAccountWithKeys
 	}
 
 	// Add MPC as a VerificationMethod for the assertion of the DID Document
-	vm, err := did.NewVerificationMethodFromBytes(doc.GetID(), ssi.ECDSASECP256K1VerificationKey2019, *controller, mtr.GetPubKey().Bytes())
+	pk, err := mtr.Wallet.CreateEcdsaFromPublicKey()
+	if err != nil {
+		mtr.triggerWalletEvent(common.WalletEvent{
+			Type:         common.WALLET_EVENT_TYPE_DID_DOCUMENT_CREATE_ERROR,
+			ErrorMessage: err.Error(),
+		})
+		return mt.CreateAccountWithKeysResponse{}, err
+	}
+	vm, err := did.NewVerificationMethod(doc.GetID(), ssi.JsonWebKey2020, *controller, pk)
```

Comment:
Looks like you're replacing the public key in the verification method with a private key which doesn't match the spec and may present security issues.

Reply:
this will pull. the public key out of the `privateKey` and use it to verify when singing the private key is still required